CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change history for ui-requests
 
 ## IN PROGRESS
+## [8.0.2](https://github.com/folio-org/ui-requests/tree/v8.0.2) (2023-05-16)
+* Create Jest/RTL test for NoteCreateRoute.js Refs: UIREQ-943
+* Added new file in fixtures folder historyData.js file for static data
+* Added NoteCreatePage function to 'ui-requests\test\jest_mock_\stripesSmartComponents.mock.js'
```

Comment:
Could you add only:
`Create Jest/RTL test for NoteCreateRoute.js. Refs: UIREQ-943.`
at the end of the "IN PROGRESS" section?